features/image-transformations/overlay-using-layers.md

```diff
@@ -0,0 +1,220 @@
+|-------------------------------------------------------------------------------------------------------------|------------------------------|
+| [w](../image-transformations/resize-crop-and-other-transformations#width-w)                                 | Width of the overlay image.  |
+| [h](../image-transformations/resize-crop-and-other-transformations#height-h)                                | Height of the overlay image. |
+| [ar](../image-transformations/resize-crop-and-other-transformations#aspect-ratio-ar)                        | Aspect ratio of the overlay image. |
+| [c](../image-transformations/resize-crop-and-other-transformations#crop-crop-modes-and-focus)               | Cropping method. Accepts `force`, `at_max`, and `at_least`. |
+| [cm](../image-transformations/resize-crop-and-other-transformations#crop-crop-modes-and-focus)              | Crop mode. Supports `extract` and `pad_resize`. |
+| [fo](../image-transformations/resize-crop-and-other-transformations#focus-fo)                               | Relative focus area used during cropping. Accepts `center`, `top`, `left`, `bottom`, `right`, `top_left`, `top_right`, `bottom_left`, and `bottom_right`. |
+| [b](../image-transformations/resize-crop-and-other-transformations#border-b)                                | This adds a border to the overlay image. It accepts two parameters - the width of the border and the color of the border in format `b-<border-width>-<hex code>` |
+| [bg](../image-transformations/resize-crop-and-other-transformations#background-color-bg)                    | It is used to specify the background color in RGB Hex Code (e.g. FF0000) or an RGBA Code (e.g. FFAABB50) that must be used for the image. If you specify an 8 character background, the last two characters must be a number between 00 and 99, which is used to indicate the opacity level of the background. `00` represents an opacity level of `0.00`, `01` represents an opacity level of `0.01`, and so on. |
+| [r](../image-transformations/resize-crop-and-other-transformations#radius-r)                                | It is used to control the radius of the corner. To get a circle or oval shape, set the value to `max`. |
+| [rt](../image-transformations/resize-crop-and-other-transformations#rotate-rt)                              | It is used to specify the degree by which the overlay image must be rotated. |
+| [t](../image-transformations/resize-crop-and-other-transformations#trim-edges-t)                            | By default, ImageKit.io trims the overlay image before overlaying it on the base image. Trimming removes similar colored pixels from the edges. To prevent the default trimming from happening, use `t-false`. **Possible values** include `true`, `false`, and integer values between `1` and `99` that specifies the threshold level for considering a particular pixel as "background". |
```

Comment:
What about zoom https://docs.imagekit.io/features/image-transformations/resize-crop-and-other-transformations#zoom-z

Reply:
added